Dockerfile

```diff
@@ -314,9 +314,9 @@ COPY --from=pythonbuild /etc/profile.d/python.sh /etc/profile.d/
 RUN chmod o+rwx ${PYENV_ROOT}
 
 # Ruby
-ARG RBENV_ROOT=/opt/rbenv/
-COPY --chown=$USERNAME:$USERNAME --from=rubybuild ${RBENV_ROOT} ${RBENV_ROOT}
-COPY --from=rubybuild /etc/profile.d/ruby.sh /etc/profile.d/
+ENV RBENV_ROOT=/opt/rbenv/
+ENV PATH=$PATH:${RBENV_ROOT}/bin:${RBENV_ROOT}/shims:${RBENV_ROOT}/plugins/ruby-install/bin
+COPY --from=ruby ${RBENV_ROOT} ${RBENV_ROOT}
```

Comment:
Similar question as in other PR: Could the change in line 319 be done in preceeding commit instead, so that 
`ruby` is taken into use instantly? (not unused)

Reply:
It will be used afterwards. I prefer stays like this, and be consistent with all other commits